kotlin/lib/src/main/kotlin/Message.kt

```diff
@@ -76,10 +76,20 @@ class Message internal constructor(token: String, options: SvixOptions) {
 /**
  * Creates a [MessageIn] with the payload already being serialized.
  *
+ * The payload is not normalized on the server (usually whitespace outside
+ * of string literals, unnecessarily escaped characters in string and such
+ * are fixed up by the server), and is not even required to be JSON.
+ *
  * @param payload Serialized message payload
  * @param contentType Content type of the payload to send as a header. Defaults to `application/json`.
  *
  * See the class documentation for details about the other parameters.
+ *
+ * # OSS server compatibility
+ *
+ * This constructor uses a mechanism that is currently only available in
+ * Svix'es cloud and on-prem enterprise products. The open source server
+ * implementation does not support it as of v1.41.0.
```

Comment:
No need to put it here. If we think it's important, let's fix OSS. Should be easy to add there.